DOC-2386: add Google Gemini integration guide (#3250)
tinymce/tinymce-docs: modules/ROOT/partials/misc/admon-ai-proxy.adoc

> IMPORTANT: The following example stores the authentication credentials in the client side integration. This is not recommended for any purpose. Instead, an alternate method for retrieving the credentials should be used such as a xref:ai-proxy.adoc[proxy server] or a server-side integration.

Reviewer: ```suggestion
IMPORTANT: For the sake of demonstration, the following examples show how to use the authentication credentials  with the API within the client side integration. This is not recommended for production purposes. We recommend only accessing the API with a xref:ai-proxy.adoc[proxy server] or implementing a server-side integration. This will prevent unauthorized access to the API.
```
Just that we are recommending that all credentials are hidden, as they shouldn't be retrieved by the client side.

> IMPORTANT: For the sake of demonstration, the following examples show how to use the authentication credentials with the API within the client side integration. This is not recommended for production purposes. It is recommended to only access the API with a xref:ai-proxy.adoc[proxy server] or by implementing a server-side integration to prevent unauthorized access to the API.

Reviewer 2: ```suggestion
IMPORTANT: The following examples are intended to show how to use the authentication credentials with the API within the client side integration. This is **not recommended** for production purposes. It is recommended to only access the API with a xref:ai-proxy.adoc[proxy server] or by implementing a server-side integration to prevent unauthorized access to the API.
```
